src/ListWithoutDouble.java

```diff
@@ -0,0 +1,34 @@
+/* Написать функцию, которая убирает одинаковые значения из списка */
+
+import java.util.ArrayList;
+
+public class ListWithoutDouble {
+
+    public static ArrayList<Integer> delDouble(ArrayList<Integer> arr) {
+        for (int i = 0; i < arr.size(); i++) {
+            for(int j = i + 1; j < arr.size(); j++){
```

Comment:
[ERROR] /Users/geoiva/Desktop/SberJava/students/maxim_rusalev/HomeWork/src/ListWithoutDouble.java:9:13: 'for' is not followed by whitespace. [WhitespaceAround]
[ERROR] /Users/geoiva/Desktop/SberJava/students/maxim_rusalev/HomeWork/src/ListWithoutDouble.java:9:13: WhitespaceAround: 'for' is not followed by whitespace. Empty blocks may only be represented as {} when not part of a multi-block statement (4.1.3) [WhitespaceAround]
[ERROR] /Users/geoiva/Desktop/SberJava/students/maxim_rusalev/HomeWork/src/ListWithoutDouble.java:9:52: '{' is not preceded with whitespace. [WhitespaceAround]
[ERROR] /Users/geoiva/Desktop/SberJava/students/maxim_rusalev/HomeWork/src/ListWithoutDouble.java:9:52: WhitespaceAround: '{' is not preceded with whitespace. [WhitespaceAround]